apaq-v1.0.yaml

```diff
@@ -0,0 +1,111 @@
+openapi: 3.0.0
+
+info:
+  title: Airport Air Quality Prediction API
+  version: "1.0"
+  description: API for predicting air quality based on airport information
+
+servers:
+  - url: /v1
+
+paths:
+  /air-quality:
+    post:
+      summary: Predict Air Quality from Airport Information
+      description: |
+        Uploads airport information and predicts air quality based on the provided data.
+        Initiates background processing for prediction.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Airport'
+      responses:
+        '201':
+          description: |
+            An air quality prediction has successfully been created and returned.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PredictionResponse'
+              example:
+                air_quality: 2.12
+        '400':
+          $ref: '#/components/responses/BadRequest'
+
+components:
+  schemas:
+    Airport:
+      type: object
+      properties:
+        altitude:
+          type: number
+          description: Altitude of the airport in feet
+        runways:
+          type: array
+          description: A list of available runways at the specified airport
+          items:
+            type: object
+            properties:
+              length:
+                type: number
+                description: Length of the runway in feet
+              surface:
+                type: string
+                description: Surface material of the runway
+        total_arrivals:
+          type: number
+          description: Total number of arriving flights in the next 24 hours.
+        total_departures:
+          type: number
+          description: Total number of departing flights in the next 24 hours.
+    
+    PredictionResponse:
+      type: object
+      properties:
+        air_quality:
+          type: number
+          description: Predicted air quality value in PM10
+
+    Error:
+      type: object
+      description: |
+        General error response body
+      properties:
+        errors:
+          description: |
+            This field MUST contain an array with at least one error model.
+          type: array
+          items:
+            $ref: '#/components/schemas/ErrorItem'
+          minItems: 1
+        status_code:
+          description: |
+            This field MAY contain the HTTP status code used for the response. Otherwise, it MUST be omitted.
+          type: integer
+      required:
+        - errors
+      additionalProperties: false
+
+    ErrorItem:
+      description: |
+        An error message.
+      type: object
+      properties:
+        code:
+
+  responses:
+    PredictionResponse:
```

Comment:
I think it might be better to identify the endpoint in the name here for extra clarity, something like `PostAirQualityResponse`?